### ISSUE TITLE
Need Documentation for res.render()

### ISSUE THREAD

**Issue by jvanalst** [balderdashy/sails-docs#380]
It's a public function but for whatever reason isn't listed along with res.view() and res.send().

Even if it's a stub saying "don't use me, use res.view()" it should probably still be there.

**Comment by alexbaumgertner** [comment]
:+1: 
I guess, its [require](https://github.com/balderdashy/sails/blob/master/lib/hooks/http/initialize.js#L68) [expressjs render](http://expressjs.com/3x/api.html#res.render) method

**Comment by rachaelshaw** [comment]
@jvanalst want to submit a PR setting up that page?

**Comment by sailsbot** [comment]
Thanks for posting, @jvanalst.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
